### PR TITLE
Format free text field

### PR DESCRIPTION
**Pull request by StupidCodeFactory**
## Description

1. Add `simple_format` in the relevant decorators.
2. move away from fixtures in favour of FactoryBot in investigation_decorator_spec.rb

## Checklist:

- [ ] Automated checks are passing locally.
- [ ] CHANGELOG updated if change is worth telling users about.
### General testing
- [ ] Test without javascript
- [ ] Test on small screen
### Accessibility testing
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable css - does content make sense and appear in a logical order?
- [ ] Passes automated checker (automated in build or manual)

### CODE REVIEW

File: psd-web/app/decorators/product_decorator.rb [hunk 1]
```diff
@@ -1,18 +1,23 @@
 class ProductDecorator < ApplicationDecorator
   delegate_all
   decorates_association :investigations
+
   def summary_list(include_batch_number: false)
     rows = [
       { key: { text: "Product name" }, value: { text: product.name } },
```
Comment (slorek) [inline psd-web/app/decorators/product_decorator.rb]: Just for my understanding: is `product` an alias for `object` inferred from the name of the class? I couldn't see documentation of this usage.

File: psd-web/spec/decorators/investigation_decorator_spec.rb [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
-    let(:investigation) { investigations(:enquiry) }
+    let(:investigation) { create(:enquiry) }
+
     context "with no product" do
       it { is_expected.to_not be_display_product_summary_list }
     end
 
     context "with products" do
-      before { investigation.products << products(:one) }
+      before { investigation.products << create(:product) }
```
Comment (slorek) [inline psd-web/spec/decorators/investigation_decorator_spec.rb]: Minor improvement might be to use `let(:products) { [create(:product)] }` here and modify the `let(:investigation)` block above.

File: psd-web/spec/factories/products.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+FactoryBot.define do
+  factory :product do
+    name { "product name" }
+    description { "product description" }
+    category { Rails.application.config.product_constants["product_category"].sample }
+    product_type { "product_type" }
+  end
+
+  factory :iphone, parent: :product do
```
Comment (slorek) [inline psd-web/spec/factories/products.rb]: I wonder if we should prepend the name of these with `product_` so it's clearer what they are.

File: psd-web/app/decorators/product_decorator.rb (hunk 1 above)
Comment (StupidCodeFactory) [reply psd-web/app/decorators/product_decorator.rb]: yep - I tend to use `object` over the inferred name from the class as it's more portable. I think I moved this code over from the investigation decorator where I would have had to anyway reference it with `product`.
I will change this to object for consistency

File: psd-web/spec/decorators/investigation_decorator_spec.rb (hunk 2 above)
Comment (StupidCodeFactory) [reply psd-web/spec/decorators/investigation_decorator_spec.rb]: so I tried to use `let(:products)` but at that point the investigation had already been created. I didn't want to do an an `object.reload` in the decorator. Adding it this way the investigation object is aware of its new products.

**Pull request merged**